lib/gmo-pg/http_resource/payload/typecast.rb

```diff
@@ -72,14 +72,16 @@ def to_attribute
               @value
             when DateTime
               Time.new(@value.year, @value.month, @value.day, @value.hour, @value.min, @value.sec)
+            when String
+              Time.strptime("#{@value} #{TIME_ZONE}", "#{FORMAT} %:z") rescue super
             else
-              Time.strptime(@value, format) rescue return super
+              super
             end
           time.to_i rescue super
```

Comment:
maybe we don't need to_i here.
just return Time type will be break nothing.

The timestamp should be explicitly changed to Time in Rails, which is quite painful.

```
[1] pry(main)> pr = Product.first
[2] pry(main)> pr.update(updated_at: 1234567890)
   (0.2ms)  BEGIN
  Product Update (1.0ms)  UPDATE "products" SET "updated_at" = $1 WHERE "products"."id" = $2  [["updated_at", nil], ["id", 4729]]
   (0.2ms)  ROLLBACK
ActiveRecord::NotNullViolation: PG::NotNullViolation: ERROR:  null value in column "updated_at" violates not-null constraint
```